<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowFlagsBox</class>
 <widget class="QDialog" name="WindowFlagsBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window flags</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rbImmediate">
           <property name="text">
            <string>Immediate</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbApply">
           <property name="text">
            <string>On Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="flagsArea" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Size</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="ShowBox" name="showBox" native="true"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="statusBarLayout"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShowBox</class>
   <extends>QWidget</extends>
   <header>showbox.h</header>
   <container>1</container>
   <slots>
    <signal>activated()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WindowFlagsBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WindowFlagsBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbImmediate</sender>
   <signal>toggled(bool)</signal>
   <receiver>WindowFlagsBox</receiver>
   <slot>setImmediate(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showBox</sender>
   <signal>activated()</signal>
   <receiver>WindowFlagsBox</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setImmediate(bool)</slot>
  <slot>reset()</slot>
 </slots>
</ui>
